openssl-dynamic/src/main/c/ssl.c

```diff
@@ -221,6 +234,243 @@ static const jint supported_ssl_opts = 0
 #endif
      | 0;
 
+static jint tcn_flush_sslbuffer_to_bytebuffer(struct TCN_bio_bytebuffer* bioUserData) {
+    jint writeAmount = TCN_MIN(bioUserData->bufferLength, bioUserData->sslWriteBufferLength) * sizeof(char);
+    jint writeChunk = bioUserData->sslWriteBufferSize - bioUserData->sslWriteBufferOffset;
+//fprintf(stderr, "tcn_flush_sslbuffer_to_bytebuffer1 bioUserData->sslWriteBufferLength %d bioUserData->sslWriteBufferOffset %d writeChunk %d writeAmount %d\n", bioUserData->sslWriteBufferLength, bioUserData->sslWriteBufferOffset, writeChunk, writeAmount);
```

Comment:
these are useful for debugging / visibility for now ... I will remove them before pulling in though.